src/doc/src/reference/profiles.md

```diff
@@ -117,7 +117,9 @@ strip = "debuginfo"
 ```
 
 Possible string values of `strip` are `"none"`, `"debuginfo"`, and `"symbols"`.
-The default is `"none"`.
+The default is `"none"` by default, and `"debuginfo"` in release profile.
```

Comment:
Similar to #13638, I don't think this is entirely correct. The default strip behavior depends on if debug is set (including overrides). 

In #13257, we somewhat intentionally did not document this, as it *should* be a completely internal matter that should be transparent to the user. It is only running `strip` to deal with the particular way the standard library is distributed.

I'm not yet entirely convinced this should be documented, since it is a relatively complicated thing to explain to the user (something they shouldn't even need to know about) which could just contribute to confusion.

Reply:
...Frankly, I don't know why there are even two options if one affects how the other is interpreted.

Reply:
In any case, I do not believe these behaviors are actually completely internal matters that are transparent to the user, as the effective behavior seems to change wildly based on the target, host, or available binaries, e.g.
- https://github.com/rust-lang/rust/issues/123114
- https://github.com/rust-lang/rust/issues/122902
- https://github.com/oxidecomputer/helios/issues/147

Perhaps we can make it one with https://github.com/rust-lang/rust/issues/123151 but it isn't now.